stage/configuration/test_directory_origin.py

```diff
@@ -464,10 +464,52 @@ def test_directory_origin_configuration_delimiter_format_type(sdc_builder, sdc_e
 @pytest.mark.parametrize('delimiter_format_type', ['CUSTOM'])
 @pytest.mark.parametrize('data_format', ['DELIMITED'])
 @pytest.mark.parametrize('enable_comments', [False, True])
-@pytest.mark.skip('Not yet implemented')
+@pytest.mark.parametrize('comment_marker', ['#'])
```

Comment:
Since, it is already a variable, may be it is worth testing with one more marker?

```suggestion
@pytest.mark.parametrize('comment_marker', ['#', '%'])
```